stdlib/SparseArrays/test/sparse.jl

```diff
@@ -2906,4 +2906,18 @@ end
     @test B ≈ mapreduce(identity, +, Matrix(A), dims=2)
 end
 
+@testset "Symmetric and Hermitian" begin
+    A = sprandn(ComplexF64, 10, 10, 0.1)
+    B = sprandn(ComplexF64, 10, 10, 0.1)
+    @test Hermitian(A + A') + B isa SparseMatrixCSC
```

Comment:
I'd suggest to explicitly _not_ make the underlying array Hermitian, so better have
```suggestion
    @test Hermitian(A) + B isa SparseMatrixCSC
```
and then also have `:U` and `:L` versions. We will also need tests for the numerical values, so something like
```julia
@test Matrix(Hermitian(A, uplo)) + Matrix(B)
```